parquet-variant/src/main/java/org/apache/parquet/variant/VariantUtil.java

```diff
@@ -848,4 +848,73 @@ static HashMap<String, Integer> getMetadataMap(ByteBuffer metadata) {
     }
     return result;
   }
+
+  /**
+   * Computes the actual size (in bytes) of the Variant value at `value[pos...]`
```

Comment:
Is this still relevant? It looks like it was copied from an instance method that expected `value` and `pos` fields.